x-pack/plugins/uptime/server/rest_api/synthetics_service/get_monitor.ts

```diff
@@ -47,25 +53,47 @@ export const getAllSyntheticsMonitorRoute: UMRestApiRouteFactory = () => ({
       search: schema.maybe(schema.string()),
     }),
   },
-  handler: async ({ request, savedObjectsClient }): Promise<any> => {
+  handler: async ({
+    request,
+    savedObjectsClient,
+    server: { encryptedSavedObjects },
+  }): Promise<any> => {
     const { perPage = 50, page, sortField, sortOrder, search } = request.query;
     // TODO: add query/filtering params
-    const {
-      saved_objects: monitors,
-      per_page: perPageT,
-      ...rest
-    } = await savedObjectsClient.find({
-      type: syntheticsMonitorType,
-      perPage,
-      page,
-      sortField,
-      sortOrder,
-      filter: search ? `${syntheticsMonitorType}.attributes.name: ${search}` : '',
-    });
-    return {
-      ...rest,
-      perPage: perPageT,
-      monitors,
-    };
+    const monitors: Array<SavedObject<SyntheticsMonitor>> = [];
+    try {
+      const encryptedClient = encryptedSavedObjects.getClient();
+      const {
+        saved_objects: encryptedMonitors,
+        per_page: perPageT,
+        ...rest
+      } = await savedObjectsClient.find({
+        type: syntheticsMonitorType,
+        perPage,
+        page,
+        sortField,
+        sortOrder,
+        filter: search ? `${syntheticsMonitorType}.attributes.name: ${search}` : '',
+      });
+      for (const monitor of encryptedMonitors) {
```

Comment:
For of loops are slower than for loops, so I could transition to a for loop here.